.vscode/settings.json

```diff
@@ -1,5 +1,17 @@
 {
   "editor.defaultFormatter": "esbenp.prettier-vscode",
+  // "[javascript]": {
```

Comment:
This seems an error to me

Reply:
yes it was showing me some warning so I commented it, I'll remove it